lawnchair/res/values/strings.xml

```diff
@@ -806,6 +808,7 @@
     <!-- Permission warnings -->
     <string name="warn_contact_permission_content">To search for contacts, grant contacts and phone permissions to Lawnchair</string>
     <string name="warn_files_permission_content">To search your files, grant storage permissions to Lawnchair</string>
+    <string name="warn_files_read_permission_content">File permission is used to render wallpaper in previews and search for your files.</string>
```

Comment:
This is mainly to keep consisting with `warn_*` string, feel free to adapt.

```suggestion
    <string name="warn_files_read_permission_content">To search your files and show wallpaper in previews, grant storage permissions to Lawnchair.</string>
```


Also

This naming (`warn_files_read_permission_content`) can be confusing, but I don't have any good recommendation at the moment. I guess `warn_files_wallpaper_permission_content` would be fine(?).